src/sagas/networkSettings.js

```diff
@@ -151,8 +154,9 @@ export function* updateNetworkSettings(action) {
 
   // - walletServiceUrl has precedence
   // - nodeUrl as fallback
+  let potentialNetwork;
   let network;
-  if (walletServiceUrl) {
+  if (walletServiceUrl && useWalletService) {
```

Comment:
_Comment unrelated to line number_

What if the user does not have the wallet-service feature flag active? We will allow him to change the URL in the dev settings but it won't do anything, as we will ignore it

Maybe we should hide the wallet-service URLs in the dev settings screen if he's not in the wallet-service

Reply:
You right! I changed the input to conditional rendering: 9d25ef9207363b3bac88a75737b20fbf6e6a331c